docs/how-to-guides/monitoring-with-prometheus-operator.md

```diff
@@ -0,0 +1,139 @@
+# Setting up monitoring on Lokomotive
+
+## Contents
+
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)
+* [Deploy Prometheus Operator](#deploy-prometheus-operator)
+  * [Configure Prometheus Operator](#configure-prometheus-operator)
+  * [Install Prometheus Operator](#install-prometheus-operator)
+* [Accessing Prometheus Operator sub-components](#accessing-prometheus-operator-sub-components)
+  * [Accessing Prometheus](#accessing-prometheus)
+  * [Accessing Alertmanager](#accessing-alertmanager)
+  * [Accessing Grafana](#accessing-grafana)
+    * [Using port forward](#using-port-forward)
+    * [Using Ingress](#using-ingress)
+
+## Introduction
+
+This guide provides installation steps of monitoring stack using prometheus-operator component and ways to access the sub-components like Prometheus, Alertmanager and Grafana.
+
+## Prerequisites
+
+- A Lokomotive cluster is accessible via `kubectl` deployed on a supported provider.
+
+<!---
+TODO: Once we have tutorials on how to deploy and configure rook or openebs, then point the following to those tutorials.
+-->
+
+- Deploy a storage provider component (rook and rook-ceph or openebs-operator and openebs-storage-class) with default storage class that can cater to the [PVC](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) requests made by prometheus-operator components Alertmanager and Prometheus.
```

Comment:
```suggestion
- A storage provider component (`rook` and `rook-ceph` or `openebs-operator` and `openebs-storage-class`) deployed with a default storage class that can provision volumes for the [PVCs](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) created by Alertmanager and Prometheus.
```

- We list prerequisites here so we shouldn't begin sentences with a verb.
- Missing articles.
- The "C" in PVC stands for "claim", which is IMO equivalent to the word "request" here. I'd say that PVCs are *created* rather than *requested*, because the PVC is itself a kind of a request.
- IMO "cater to" doesn't match the tone of a technical document. I've suggested an alternative phrasing.
- I'd backtick any Lokomotive component names to highlight that we're talking about specific components which the user has to deploy.